sormas-backend/src/main/java/de/symeda/sormas/backend/action/ActionService.java

```diff
@@ -191,6 +192,70 @@ public Predicate createUserFilter(CriteriaBuilder cb, CriteriaQuery cq, From<?,
 		return QueryHelper.getResultList(em, cq, first, max);
 	}
 
+	public List<Action> getActionList(ActionCriteria actionCriteria, Integer first, Integer max, List<SortProperty> sortProperties) {
```

Comment:
Please unify this method with the existing `getActionList` method (i.e., remove the code from the existing method and let it call this method with null as sortProperties) to avoid duplicate code